website/content/api-docs/agent/service.mdx

```diff
@@ -693,10 +693,9 @@ For the `Connect` field, the parameters are:
   If this is true, then service mesh proxies, DNS queries, etc. will be able to
   service discover this service.
 - `Proxy` `(Proxy: nil)` -
-  [**Deprecated**](/consul/docs/connect/proxies/managed-deprecated) Specifies that
+  [**Deprecated**] Specifies that
```

Comment:
```suggestion
  [**Deprecated**] Specifies that
```
According to the MD style guide, we should add \<sup>Deprecated\</sup> to the `Proxy` flag.